ckan-backend-dev/src/ckanext-wri/ckanext/wri/logic/action/get.py

```diff
@@ -577,3 +602,35 @@ def dataset_release_notes(context: Context, data_dict: DataDict):
     results = q.all()
 
     return results
+
+
+@logic.side_effect_free
+def dashboard_activity_listv2(context: Context, data_dict: DataDict):
+    # get_action for dashboard_activity_list
+    model = context["model"]
+    results = get_action("dashboard_activity_list")(context, data_dict)
```

Comment:
same thing as before, if you keep a dictionary of users, you can decrease the amount of calls per user to only one, so if 30 activities are from the same user, you make only one call for the db